BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -137,9 +160,19 @@ def listen_for_light_change_events(self, interval: float, callback: Callable):
                             callback(device, sgroup["name"], timestamp, new_status)
 
     def stop_polling(self):
+        """ To stop polling """
```

Comment:
Docstring should provide more information than is already available by looking at the method signature.

I would say "Stops polling the API after the current polling cycle is completed. It may take up to `interval` seconds for the polling thread to finish".